_delphi_utils_python/delphi_utils/utils.py

```diff
@@ -96,4 +96,30 @@ def transfer_files():
     files_to_export = os.listdir(export_dir)
     for file_name in files_to_export:
         if file_name.endswith(".csv") or file_name.endswith(".CSV"):
-            move(os.path.join(export_dir, file_name), delivery_dir)
+            move(os.path.join(export_dir, file_name), os.path.join(delivery_dir, file_name))
+
+def delete_move_files():
+    """Delete csv files in export-dir under certain conditions.
+
+    1. Delivery-dir is specified (aka we are only deleting files produced by the run
+    2. If validation-failures-dir is specified, move failures there instead
+    """
```

Comment:
Changing the wording a bit for extra precision.

```suggestion
    """Delete or move output files depending on dir settings provided in params.

    1. Delete files in export-dir if delivery-dir is specified and is different from export_dir (aka only delete files produced by the most recent run)
    2. If validation-failures-dir is specified, move failed files there instead
    """
```